lib/utils/testem-events.js

```diff
@@ -121,6 +121,17 @@ class TestemEvents {
     }
   }
 
+  /**
+   * Record the module run details to the stateManager
+   *
+   * @param {*} moduleName
+   * @param {*} totalAssertionNum
+   * @param {*} moduleTotalDuration
+   */
+  recordModuleRunDetail(moduleName, totalAssertionNum, moduleTotalDuration) {
```

Comment:
as suggested above, could be renamed & simplified to recordModuleMetadata(metadata){